assets/Product.wxs

```diff
@@ -10,14 +10,16 @@
   <?define ProductName = "$(env.ProductName)" ?>
   <?define ProductGuid = "$(env.ProductGuid)" ?>
   <?define ProductVersion = "$(env.ProductVersion)" ?>
+  <?define ProductSemanticVersion = "$(env.ProductSemanticVersion)" ?>
   <?define ProductVersionWithName = "$(var.ProductName)_$(var.ProductVersion)"?>
+  <?define ProductSemanticVersionWithName = "$(var.ProductName)_$(env.ProductSemanticVersion)"?>
```

Comment:
Added commit  to use hyphen where possible. Note: Wix does not like hyphen internally in the config. But all user facing names use hyphen consistently

![image](https://cloud.githubusercontent.com/assets/11727200/24467777/b0e65948-146a-11e7-9f62-e57c96b276d9.png)

![image](https://cloud.githubusercontent.com/assets/11727200/24467784/b6a3878e-146a-11e7-99bd-8762c8904a7d.png)

![image](https://cloud.githubusercontent.com/assets/11727200/24467803/c4b4932c-146a-11e7-83ff-e0bbc8438def.png)